### DISCUSSION TITLE
how to open the mission in the editor???

### DISCUSSION THREAD

**Discussion by snowymidnights** [Vdauphin/HeartsAndMinds#1525] [Question and answer]
hello, as the title states I'm trying to replace the vanilla vehicles with RHS vehicles but I'm not even sure how? apparently you have to open up the missions sqm but I'm not exactly sure where you find that or how you even open it. help would be appreciated :)

**Comment by Vdauphin** [comment]
Hello,
You can join the A3 Official Discord for help: https://discord.gg/arma
Cheers
